package.json

```diff
@@ -41,6 +41,8 @@
     "@types/react": "^17.0.40",
     "@types/react-dom": "17.0.13",
     "@types/web": "^0.0.55",
+    "glob": "^7.2.0",
```

Comment:
Why not move these two dependencies to the `packages/scripts`?

Reply:
Will do.